tools/cli/help.txt

```diff
@@ -440,26 +440,15 @@ commands can be accessed by pressing the up arrow.
 
 
 >>> mongo
-Connect to the Mongo database for the specified site.
-Usage: meteor mongo [--url]
```

Comment:
I think we should leave the `--url` / `-U` docs in place here. Using `meteor mongo --url` is incredibly handy when you want to get Meteor's current full mongo URL, to integrate that Mongo instance with something else locally (like when developing a custom app that feeds data into Mongo directly).

Reply:
Agree, we should probably keep this unless there's a pressing reason to remove it.  Removing it could cause an issue if anyone was using this in an external script right now (i.e. `mongo $(meteor mongo --url)`).

Reply:
Also, again to clarify, this is something that I want to fix because I think it's incredibly useful. I just don't want inaccurate documentation as to what it currently does

Reply:
Well, technically, the code is still in place to _try_ to connect to a remote Mongo if a `site` is passed to the `meteor mongo` URL.  That code can be found here:

https://github.com/meteor/meteor/blob/9b056f5e911513fb3df716bd524d504cceb863f4/tools/cli/commands.js#L1254-L1264

We should definitely address that!